tests/pyccel/scripts/arrays_view.py

```diff
@@ -39,8 +39,24 @@ def array_view_negative_variable_step():
         for j in range(5):
             print(b[i][j])
 
+@allow_negative_index('a')
+def array_view_negative_var():
+    a = np.array([[1, 2, 3], [4, 5, 6], [7, 9, 5]])
+    v = -1
+    x = a[v,1:]
+    for i in range(2):
+        print(x[i])
+
+def array_view_positive_literal():
```

Comment:
It seems like this test isn't used. Was that done on purpose?

Reply:
Oups, i forgot to call the function